### ISSUE TITLE
HTML is escaped in featured image caption

### ISSUE THREAD

**Issue by pontikis** [oceanwp/oceanwp#210]
Congratulations for your excellent software.

Caption in featured image is very useful to display image Credits (with a link to image creator)

Even if caption with HTML is displayed correctly in blocked image inside a post, **it is escaped in featured images**. So, you cannot use HTML in featured images caption.

So, regarding these files

- wp-content/themes/oceanwp/partials/single/media/blog-single.php
- wp-content/themes/oceanwp/partials/single/media/blog-single-link.php
- wp-content/themes/oceanwp/partials/entry/media/blog-entry.php
- wp-content/themes/oceanwp/partials/entry/media/blog-entry-link.php

I suppose that this 

```php
	// Caption
	if ( $caption ) { ?>
		<div class="thumbnail-caption">
			<?php echo esc_attr( $caption ); ?>
		</div>
	<?php
	} ?>
```

should be replaced by

```php
	// Caption
	if ( $caption ) { ?>
		<div class="thumbnail-caption">
			<?php echo wp_kses_post( $caption ); ?>
		</div>
	<?php
	} ?>
```

Thank you!

**Comment by easterncoder** [comment]
Experiencing the same issue. Would love to attribute the image source for my featured photos but `esc_attr` strips out the html.

Confirming that @pontikis suggestion above works.

**Comment by tdtgit** [comment]
Not sure why it's not fixed so I create a new PR #227

**Comment by eramits** [comment]
We will add the fix in the upcoming updates of the themes. Thank you @pontikis and @tdtgit for the fix.
